src/secrets/CommandEdit.ts

```diff
@@ -1,8 +1,9 @@
 import type PolykeyClient from 'polykey/dist/PolykeyClient';
 import fs from 'fs';
 import path from 'path';
-import * as errors from '../errors';
+import * as vaultsErrors from 'polykey/dist/vaults/errors';
 import CommandPolykey from '../CommandPolykey';
```

Comment:
Something like this needs to be imported asynchronously in the command. Have a look at `src/nodes/CommandFind.ts:26` for reference.

We do this to only import what we need when we need it. Otherwise we end up importing all of polykey regardless of the command we're running.